test/layout/test_page_background.py

```diff
@@ -0,0 +1,102 @@
+from fpdf import FPDF, drawing
+from PIL import Image
+import io
+from pathlib import Path
+from test.conftest import assert_pdf_equal
+from datetime import datetime
+
+
+HERE = Path(__file__).resolve().parent
+imgpath = str(HERE / "../image/image_types/insert_images_insert_jpg.jpg")
+
+
+def rgb_tuple_multi_page(pdf, fill_set=None):
+    """
+    sets the background using a tuple representing an RGB color and adds a page with text and a small rectangle,
+    then writes multiline-text to trigger a page break to test if the background color is retained
+    """
+    pdf.set_page_background((170, 200, 100))
+    pdf.add_page()
+    pdf.rect(20, 20, 60, 60, style="F")
+    pdf.cell(0, 30, "RGB tuple", align="R", border=1)
+    pdf.ln()
+    if fill_set:
+        pdf.cell(0, 30, "fill color set", align="R", fill=True)
+    else:
+        pdf.cell(0, 30, "default fill color", align="R")
+    pdf.ln()
+    pdf.multi_cell(0, pdf.h / 2, "PAGE 1\nPAGE 2")
```

Comment:
Could you please call `assert_pdf_equal(pdf, HERE / "rgb_tuple_multi_page.pdf", tmp_path, generate=True)` at the end of this code block? This will ensure the PDF you built in this function will be checked against a reference PDF file, stored in the `git` repository (it must be `git added`).

The `generate=True` argument is very useful to write new tests as it will generate a reference PDF file.
Once you'll be happy with the PDF file generated, you will just have to remove `generate=True` and you'll get a fine non-regression unit test.